services/provider/server/server.go

```diff
@@ -367,6 +375,40 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 		}),
 	})
 
+	// Fetch noobaa remote secret and management address and append to extResources
+	noobaaOperatorSecret := &v1.Secret{}
+	noobaaOperatorSecretName := consumerResource.Name
+	err = s.client.Get(ctx, types.NamespacedName{Name: noobaaOperatorSecretName, Namespace: s.namespace}, noobaaOperatorSecret)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get %s secret. %v", noobaaOperatorSecretName, err)
+	}
+
+	authToken, ok := noobaaOperatorSecret.Data["auth_token"]
+	if !ok || len(authToken) == 0 {
+		return nil, fmt.Errorf("auth_token not found in %s secret", noobaaOperatorSecretName)
+	}
+
+	noobaMgmtRoute := &routev1.Route{}
+	err = s.client.Get(ctx, types.NamespacedName{Name: "noobaa-mgmt", Namespace: s.namespace}, noobaMgmtRoute)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get noobaa-mgmt route. %v", err)
+	}
+	if noobaMgmtRoute.Status.Ingress == nil || len(noobaMgmtRoute.Status.Ingress) == 0 {
+		return nil, fmt.Errorf("no Ingress available in noobaa-mgmt route")
+	}
+
+	noobaaMgmtAddress := noobaMgmtRoute.Status.Ingress[0].Host
+	if noobaaMgmtAddress == "" {
+		return nil, fmt.Errorf("no Host found in noobaa-mgmt route Ingress")
+	}
+	extR = append(extR, &pb.ExternalResource{
+		Name: consumerResource.Name,
```

Comment:
We are sending the storageConsumer Name here but on [client-operator](https://github.com/red-hat-storage/ocs-client-operator/pull/176/files#diff-67ad3ddf7a24ad71e9510d7547db6201545f32798543ef7d0752cc01b7fe6be7R138) side we are checking the storgeClient Name this will not create the secret on the client side

Reply:
Yes. changed to consumer Name here